packages/core/src/RenderingEngine/helpers/cpuFallback/rendering/storedColorPixelDataToCanvasImageData.ts

```diff
@@ -38,8 +38,7 @@ export default function (
       canvasImageDataData[canvasImageDataIndex++] =
         lut[pixelData[storedPixelDataIndex++] + -minPixelValue]; // Green
       canvasImageDataData[canvasImageDataIndex] =
-        lut[pixelData[storedPixelDataIndex] + -minPixelValue]; // Blue
-      storedPixelDataIndex += 2;
+        lut[pixelData[storedPixelDataIndex++] + -minPixelValue]; // Blue
```

Comment:
The position was setup for RGB to RGBA copying, but in fact it is RGB to RGB copy.